.gitsubprojects

```diff
@@ -3,5 +3,4 @@ git_subproject(vmmlib https://github.com/Eyescale/vmmlib.git 639ccfd)
 git_subproject(zeq https://github.com/HBPVis/zeq.git a690304)
 git_subproject(Lunchbox https://github.com/Eyescale/Lunchbox.git 7d3b9b2)
 git_subproject(dash https://github.com/BlueBrain/dash.git 6e05142)
-git_subproject(Collage https://github.com/Eyescale/Collage.git f615b9c)
```

Comment:
Well, in the other pull request I argued to keep it *partly* because it was leaking, but I didn't imply that we could remove non-leaking dependencies. I am not sure of the benefits of this approach you propose. In fact, if you remove all non-leaking dependencies, you may end up with no subprojects at all... The problem is more complex, because there are often multiple conflicting inclusions. Look at Lunchbox for instance: all of dash, collage and equalizer require it, no matter in what order you put them it will override some other software's version. Also, as I had argued here: https://github.com/BlueBrain/Monsteer/pull/13#discussion-diff-45599720 if you remove collage, and suddenly livre needs a new version of collage, you have to update first equalizer, then update equalizer here. This may take some time until the changes "bubbles up" to the downstream projects, vs just updating livre directly.